src/components/Download.tsx

```diff
@@ -1,10 +1,232 @@
+    a.click();
+    URL.revokeObjectURL(url);
+    document.body.removeChild(a);
+  };
+
   return (
-    <div className="flex flex-col items-center">
-      <h1>Download</h1>
-      <NavigationButton label="Back - Value Annotation" viewToNavigateTo="valueAnnotation" />
+    <div className="flex flex-col items-center p-6" data-cy="download">
+      <Typography variant="h4" className="mb-6 font-bold">
+        Download
+      </Typography>
+
+      {!schemaValid && (
+        <Alert
+          severity="error"
+          data-cy="incomplete-annotations-alert"
+          className="mb-6 w-full max-w-2xl"
+        >
+          <Typography variant="h6" className="mb-2 font-bold">
+            Incomplete Annotations
+          </Typography>
+          <Typography variant="body1" className="mb-2">
+            There are incomplete annotations for columns that were mapped to variables. You can go
+            back and complete these annotations before you download the .json data dictionary.
+          </Typography>
+          <Typography variant="body1" className="mb-2">
+            The following columns have incomplete annotations:
+          </Typography>
+          <List className="list-disc pl-6" data-cy="incomplete-annotations-list">
+            {schemaErrors.map((column) => (
+              <ListItem key={column} className="list-item">
+                <Typography variant="body1">{column}</Typography>
+              </ListItem>
+            ))}
+          </List>
+          <Typography variant="body1" className="mb-2">
+            <b>NOTE</b>: If you download the .json data dictionary without completing the
+            annotations, the data dictionary will not work with subsequent neurobagel steps.
+          </Typography>
+          <Typography variant="body1" className="mb-2">
+            <b>
+              You will not be able to load the data dictionary into the annotation tool to change
+              mistakes later.
+            </b>
+          </Typography>
+        </Alert>
+      )}
+
+      {schemaValid && (
+        <Alert
+          data-cy="complete-annotations-alert"
+          severity="success"
+          className="mb-6 w-full max-w-2xl"
+        >
+          <div className="flex items-center gap-2">
+            <img src={emoji} alt="bagel confetti" className="h-10 w-10" />
+            <Typography variant="h4" className="font-bold">
+              Congratulations!
+            </Typography>
+          </div>
+          <Typography variant="body1">
+            You have successfully created a{' '}
+            <Link
+              href="https://neurobagel.org/data_models/dictionaries/"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="hover:underline"
+            >
+              neurobagel annotated .json data dictionary
+            </Link>
+            .
+          </Typography>
+        </Alert>
+      )}
+
+      <div className="w-full max-w-2xl">
```

Comment:
this seems like it could be a good generic "alert" component that we just pass the relevant content / props to, no?

Reply:
Yea it could be a good candidate for a generic component but I'd say we wait on that until we know where it be used/reused and therefore how generic it needs to be: #68